third_party/proto/buf.yaml

```diff
@@ -0,0 +1,26 @@
+# Generated by "buf config migrate-v1beta1". Edit as necessary, and
```

Comment:
This buf.yaml is used to reduce the number of third party protos needed in the repo. The problem boils down to three groups of files: Our needed protos, the protos we need as dependencies, and the protos needed by those dependencies + whatever dependencies those dependencies have (ad infinitum).

Our protos compile just fine if they have their bare minimum dependencies. `make proto-lint` will unfortunately verify the dependencies' dependencies and this buf.yaml tells it to ignore everything we have in the third_party folder. We can't reasonably make changes to those protos anyway, so this reduces headache without adding concerns.

The contents of this file ONLY modify `make proto-lint`, not the breaking test nor the compilation.